new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java

```diff
@@ -133,6 +135,44 @@ public void beforeAll(ExtensionContext context) {
         assertFalse(operatorImage.isEmpty(), "Image name can not be empty");
         assertTrue(Operator.buildImage(operatorImage), "docker build failed for Operator");
 
+        // docker login to OCR if OCR_USERNAME and OCR_PASSWORD is provided in env var
+        if (!OCR_USERNAME.equals(REPO_DUMMY_VALUE)) {
+          withStandardRetryPolicy
+              .conditionEvaluationListener(
+                  condition -> logger.info("Waiting for docker login to be successful"
+                          + "(elapsed time {0} ms, remaining time {1} ms)",
+                      condition.getElapsedTimeInMS(),
+                      condition.getRemainingTimeInMS()))
+              .until(() -> dockerLogin(OCR_REGISTRY, OCR_USERNAME, OCR_PASSWORD));
+        }
+
+        // The following code is for pulling WLS images if running tests in Kind cluster
+        if (KIND_REPO != null) {
+          // We can't figure out why the kind clusters can't pull images from OCR using the image pull secret. There
```

Comment:
Do not put open ended question in description on publicly available class.

Reply:
This was added by Ryan. @xiancao please change the wording to 

kind clusters can't pull the images from OCR using the image pull secret....